_docs/_latest/develop/dev/inventory.md

```diff
@@ -58,12 +61,13 @@ in Inventory.
                     yield FACTORIES['image'].create_new(data)
 ```
 
-In order to do the casting, we need to create a resource class for foo. This
-allows us to access the `id` and `type`. If the resource has no provided `id`,
-we must synthesize our own. See other existing `key()`, to see how to create
-a synthetic key.
-
+To complete the casting, edit
 [google/cloud/forseti/services/inventory/base/resources.py](https://github.com/GoogleCloudPlatform/forseti-security/tree/stable/google/cloud/forseti/services/inventory/base/resources.py)
+to create a resource class for foo. This allows you to access the `id` and `type`.
+If the resource doesn't have a provided `id`, you'll have to synthesize one. For
+details to create a synthetic key, see an existing `key()`.
```

Comment:
@blueandgold  Where do I find an existing key?

Reply:
Updated with details on how to find how other synthetic keys are generated.